impala/dbapi.py

```diff
@@ -192,7 +192,7 @@ def connect(host='localhost', port=21050, database=None, timeout=None,
         http_cookie_names = auth_cookie_names
     elif http_cookie_names is None:
         # Set default value as the list of HTTP cookie names used by Impala and Hive.
```

Comment:
Could you update the comments for three new Knox cookies? Are they used for session management? For cookie-based authentication, cookie name must end with '.auth'.

Reply:
@wzhou-code  Thank you for the review, I have updated the comments and moved them to the end. Those cookies are indeed used by Knox and are part of defaulg cookies list for ODBC Driver too.
Ref [Page 98]: https://docs.cloudera.com/documentation/other/connectors/hive-odbc/2-7-0/Cloudera-ODBC-Driver-for-Apache-Hive-Install-Guide.pdf

<img width="691" alt="Screenshot 2024-01-03 at 1 08 30 PM" src="https://github.com/cloudera/impyla/assets/24653557/13ab000b-3e95-4055-a12b-908e04bb4b21">